### DISCUSSION TITLE
Beeper

### DISCUSSION THREAD

**Discussion by harukayamazaki** [catppuccin/catppuccin#2205] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

Theme uses CSS and some examples can found here https://github.com/beeper/themes

### Any additional comments?

More information about Beeper can be found here https://www.beeper.com/ and is currently on a invite system. You can use my code to get you started Beeper: refer.beeper.com/Gwbux9

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by harukayamazaki** [comment]
Beeper can be further customized with some modifications https://gist.github.com/0xdevalias/3d2f5a861335cc1277b21a29d1285cfe#devalias-beeper-css-hacks

**Comment by harukayamazaki** [comment]
![catppuccinmochabeepertheme](https://github.com/catppuccin/catppuccin/assets/60632993/8927f7e3-94b9-4ad8-b540-39ace961b71c)
I started some small progress and used this as the base https://github.com/Madelena/metrology-for-beeper
[Altered-Metrology.txt](https://github.com/catppuccin/catppuccin/files/13573564/Altered-Metrology.txt)

**Comment by harukayamazaki** [comment]
I stopped all progress after beeper kept having issues. I went with bluebubbles and opt to self host imessage. It has built in theme management.

**Comment by PoorPocketsMcNewHold** [comment]
![image](https://github.com/user-attachments/assets/f3d784f4-694a-4565-8e3d-3caa4e638ceb)
![image](https://github.com/user-attachments/assets/3b947846-500d-49b3-b1e6-b84e34854ff0)
https://gist.github.com/PoorPocketsMcNewHold/4ae6d7052208d01103159a0617d77316
Here's another theme attempt, using https://github.com/josselinonduty/beeper-theme-solarized and https://gist.github.com/0xdevalias/3d2f5a861335cc1277b21a29d1285cfe as a base.
